z/zellij/monitoring.yml

```diff
@@ -0,0 +1,2 @@
+releases:
+  id:
```

Comment:
Drop a [300087](https://release-monitoring.org/project/300087/) here :slightly_smiling_face:

Reply:
Done, what does this relate to?